math/dec_test.go

```diff
@@ -668,11 +668,15 @@ func BenchmarkLegacyQuoMut(b *testing.B) {
 
 func BenchmarkLegacyQuoTruncateMut(b *testing.B) {
 	b1 := math.LegacyNewDec(17e2 + 8371)
+	baseArr := make([]math.LegacyDec, b.N)
+	for i := 0; i < b.N; i++ {
+		baseArr[i] = b1.Clone()
+	}
 	b2 := math.LegacyNewDec(4371)
 	b.ReportAllocs()
 	b.ResetTimer()
 	for i := 0; i < b.N; i++ {
-		sink = b1.QuoTruncateMut(b2)
+		sink = baseArr[i].QuoTruncateMut(b2)
```

Comment:
good fix. Do you want to add more number examples to support your cases? You could run them via 
`b.Run("", func(b *testing.B) {})` like table tests